src/main/java/entity/CorporatePage.java

```diff
@@ -22,11 +33,17 @@ public String getCompanyInfo() {
         return companyInfo;
     }
 
-    public CorporateRep getPageManager() {
-        return pageManager;
+    public ArrayList<Internship> getInternshipsArray() { return this.internshipsArray; }
+
+    public void setCompanyName(String companyName){
+        this.companyName = companyName;
     }
 
-    public int getCompanyPageId() {
-        return companyPageId;
+    public void setCompanyInfo(String companyInfo){
+        this.companyInfo = companyInfo;
     }
+
+    public void addInternship(Internship internship) { this.internshipsArray.add(internship); }
```

Comment:
This is purely style, but I don't think that including the word Arrays in variable names is that common; could just rename to "interships" or something else

Reply:
changed